src/app/key-concepts/natural-capital-and-biodiversity/page.tsx

```diff
@@ -0,0 +1,214 @@
+                <p className="font-bold">Example of natural capital, biodiversity and beyond</p>
+                <p>
+                  Ecosystem services such as pollination, pest control, and the production of food
+                  and fiber depend directly on biodiversity.
+                </p>
+              </div>
+            </div>
+          </div>
+          <div className="flex flex-col gap-6 self-start text-grey-800 max-lg:pb-5 lg:max-w-[799px]">
+            <div className="text-2xl tracking-tight lg:max-w-[737px] lg:text-4xl">
+              We need to protect both for sustainability for two reasons:
+            </div>
+          </div>
+          <div className="flex w-full items-center justify-center">
+            <div className="flex flex-col items-center gap-10 lg:w-[559px] lg:gap-20">
+              <div className="flex w-full flex-col gap-3 lg:gap-5">
+                <div className="text-[52px] leading-[48px] text-pink lg:text-5xl">01</div>
+                <div className="text-2xl lg:text-4xl">Resilience to environmental changes</div>
+                <div>
+                  <p>
+                    Protecting biodiversity is essential for maintaining natural capital and
+                    ensuring the resilience of ecosystems to environmental changes.
+                  </p>
+                  <p>
+                    Resilience refers to the ability of a system to withstand and/or recover from
+                    shocks and pressures, such as drought, fire, flood or disease.
+                  </p>
+                </div>
+              </div>
+              <div className="flex w-full gap-6">
+                <div className="w-1.5 bg-pink" />
+                <div className="max-w-[510px] grow basis-0">
+                  Sustainable management practices should consider both the conservation of
+                  biodiversity and the sustainable use of natural resources to promote long-term
+                  environmental and economic sustainability.
+                </div>
+              </div>
+              <div className="flex w-full flex-col gap-3 lg:gap-5">
+                <div className="text-[52px] leading-[48px] text-pink lg:text-5xl">02</div>
+                <div className="text-2xl lg:text-4xl">Well-defined and measurable attributes</div>
+                <div>
+                  Biodiversity has well-defined and measurable attributes (e.g., the number of
+                  species present, the amount of genetic variability in a population, or the
+                  diversity of species that perform the same or similar functions) that are
+                  indicators of the resilience of ecosystem assets.
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div className="lg:ox-0 -mx-6 border-t border-dashed border-grey-800 px-6 py-6 lg:mx-0 lg:pt-20">
+          <div className="mx-auto flex max-w-7xl flex-wrap gap-[43px]">
```

Comment:
Can you use similar classes as on the homepage? https://github.com/Vizzuality/natural-capital-primer/blob/b91e0428a4365fa75e42b0f7264c41ff3f40cf56/src/app/page.tsx#L178